index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
@marco-c It's a bit complicated, but easier if you remember what VAPID is supposed to be. VAPID is basically a way for the feed provider to offer information about what the feed is. "Sub" is the critical item, and should be an email that Push Service providers can use to contact someone from the feed provider side. So it's the address you send the "Hey, we noticed a problem..." or "HOLY CRAP WOULD YOU CUT IT OUT" sort of messages. Since it's more than possible that one subscription service could provide lots of different subscriptions (Think of a proxy service like OneSignal or Feedly or someone) "Aud" is there to identify the individual feed. 

For that, it could be anything like the simple host origin, or a proxied ID URI, or honestly anything that lets the two operational teams quickly identify the feed that's having a problem. 

VAPID is basically an agreement between the feed provider and the push service. I'll note that the feed provider could also be a jerk and load VAPID with bogus information. In that case, the Push Service is completely within it's rights to deny the feed, ignore it, or rate limit it as much as they desire. VAPID is there pretty much for the benefit of the subscription provider.

So, in short: Yes, "aud" should contain a URL. What that URL actually is should be up to the subscription provider, but it'd be darn useful if it was meaningful in some fashion.

Now, all that said, I'll go look over the spec and see if we need to modify things.